### DISCUSSION TITLE
Disk Pressure due to high allocation

### DISCUSSION THREAD

**Discussion by wopl** [longhorn/longhorn#6785] [Q&A]
## Question
> Disk Pressure due to high allocation

## Environment

 - Longhorn version: 1.40
 - Kubernetes version: 1.26 (k3s)
 - Node config
   - Ubuntu 22.04

## Additional context

For some unknown reason I get some disk-pressure. For most applications I distribute 3 Replicas across the three nodes. Therefore Disk space should be sufficient (as seen on the other nodes).
The shown 309 allocations cannot be found on disk at /home/longhorn

Could anyone guide me to figure out the root reason and potentially garbage collect this issue?
![longhorn](https://github.com/longhorn/longhorn/assets/6441547/2f1bd081-364a-4e15-b5df-1f020a35ddb8)

**Comment by derekbit** [comment]
Not sure if it is related to https://github.com/longhorn/longhorn/issues/5762.
Can you check the ticket?

If yes, you can apply the workaround in https://github.com/longhorn/longhorn/issues/5762 and https://github.com/longhorn/longhorn/issues/6121#issuecomment-1590441293.

**Comment by wopl** [comment]
I applied the recommendations from #5762.

This solved my Issue, but a warning:
`kubectl -n longhorn-system get replica | grep stopped | awk '{ print $1 }' | xargs kubectl -n longhorn-system delete replica`
will delete volumes from applications that are currently not running (Pod scaled to 0); So in my case 4 Applications were lost. No issue here, but could have been more serious.

Will upgrade to Longhorn 1.4.2 soon. So the issue will not re-occur?
Many thanks for the quick answer.

**Comment by derekbit** [comment]
> Will upgrade to Longhorn 1.4.2 soon. So the issue will not re-occur?

If you apply the workaround in https://github.com/longhorn/longhorn/issues/5762#issue-1669167403, the issue should not happen when upgrading to v1.4.2.
